packages/govuk-frontend/tasks/build/package.unit.test.mjs

```diff
@@ -109,8 +109,8 @@ describe('packages/govuk-frontend/dist/', () => {
 
       // Only sass package entry is compiled to minified CSS bundle
       .flatMap(
-        mapPathTo(['**/govuk/all.scss'], ({ dir: requirePath }) => [
-          join(requirePath, 'all.scss'),
+        mapPathTo(['**/govuk/index.scss'], ({ dir: requirePath }) => [
+          join(requirePath, 'index.scss'),
```

Comment:
**question** What are your thoughts on duplicating the tests while we have both `index.scss` and `all.scss`, as a safety net that the deprecated file still works as intended?